app/models/archived_background.rb

```diff
@@ -1,2 +1,5 @@
 class ArchivedBackground < ApplicationRecord
```

Comment:
it would be nice to have some class comments here @cdebourcy . when should we be archiving backgrounds?